application/src/main/java/org/togetherjava/tjbot/features/moderation/TransferQuestionCommand.java

```diff
@@ -189,7 +191,7 @@ private RestAction<Message> dmUser(MessageChannelUnion sourceChannel, ForumPost
                 """
                         Hello%s 👋 You have asked a question in the wrong channel%s. Not a big deal, but none of the experts who could help you are reading your question there 🙁
 
-                        Your question has been automatically transferred to %s, please continue there, thank you 👍
+                        Your question has been automatically transferred to %s , please continue there, thank you 👍
```

Comment:
the extra space is confusing and the next guy seeing this code will remove it again.

was this to get rid of the auto-preview? then put it into an extra variable with a proper name and perhaps a comment. and move it down to where you put in the jumpurl, not up here.
so:

```java
// Prevents Discord creating a distracting auto-preview for the link
String jumpUrlSuffix = " ";

messageTemplate.formatted(..., forumPost.message.getJumpUrl() + jumpUrlSuffix);
```